### DISCUSSION TITLE
Classes missing in the GitHub source code

### DISCUSSION THREAD

**Discussion by DaniOnrubia** [rabbitmq/rabbitmq-java-client#1047] [General]
### Describe the bug

Hi,

Two classes are missing in the source code. That makes it impossible to me to download it and compile it by myself.

The classes missing are
- com.rabbitmq.client.AMQP
- com.rabbitmq.client.impl.AMQImpl

Kind Regards,
Dani

### Reproduction steps

1. Download the source code from GitHub.
2. Unzip it and open the project it with any IDE
3. More than 1 thousand errors due to missing classes.

### Expected behavior

I expect to ble able to download the GitHub code and compile it by myself with the latest patches and enhancements.

**Comment by michaelklishin** [comment]
Those classes are code-generated. This has been the case since 2006-2007 and will not change.

[codegen.py](https://github.com/rabbitmq/rabbitmq-java-client/blob/main/codegen.py) in the repo
root. It is driven by Maven, so you don't need to run it.

At least some modern IDEs (e.g. IDEA) allow for downloads of source JARs for dependencies. Others
can import a Maven project, build it, and then discover/index the generated files.
